contracts/erc20-gateway/ERC20Cogateway.sol

```diff
@@ -384,6 +402,11 @@ contract ERC20Cogateway is
             );
 
             utilityTokens[_valueToken] = utilityToken_;
+
+            emit UtilityTokenCreated(
```

Comment:
Please, update documentation and add post condition with event (to be consistent with other functions in the contract).

Reply:
Yes. Added \post condition for `UtilityTokenCreated` event.